server/users/userController.js

```diff
@@ -0,0 +1,44 @@
+var User = require('./userModel.js');
+Q = require('q');
+
+var findUser = Q.nbind(User.findOne, User);
+var createUser = Q.nbind(User.create, User);
+
+module.exports = {
+
+  createOrFindOne: function (profile) {
+    var fbId = profile.id;
+    var name = profile.displayName;
+    var picture = profile.photos[0].value;
+    var friends = profile._json.friends.data.map(function(friend) {
+      return {fbId: friend.id}; 
+    });
+
+      findUser({fbId: fbId})
```

Comment:
Does this (line 17) need to be un-indented to be at the same level as createOrFindOne? Also, indent lines 10-14?